src/test/java/org/apache/commons/lang3/builder/DiffBuilderTest.java

```diff
@@ -321,15 +346,50 @@ public void testObject() throws Exception {
     
     @Test
     public void testObjectsEqual() throws Exception {        
-        final TypeTestClass class1 = new TypeTestClass();
-        final TypeTestClass class2 = new TypeTestClass();
-        class1.objectField = "Some string";
-        class2.objectField = "Some string";
-        final DiffResult list = class1.diff(class2);
-        assertEquals(0, list.getNumberOfDiffs());
+        // Same (left == right).
```

Comment:
Would be better to create separate test methods for the four blocks you're proposing here.